### ISSUE TITLE
Standardize customer export

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/Easy-Digital-Downloads#3434]
Right now the data that gets exported for the Customer export changes based on which options you choose.

For example, if you choose to export customers of a specific download, the data structure is different than if you export customers of all downloads. This is silly.

Once #2999 is complete, we can standardize this so it's always the same.

**Comment by pippinsplugins** [comment]
Payments and customers are done.

**Comment by pippinsplugins** [comment]
Done in https://github.com/easydigitaldownloads/Easy-Digital-Downloads/pull/3472

**Comment by pippinsplugins** [comment]
Merged into `release/2.4`
